April23/update-cycle.md

```diff
@@ -0,0 +1,74 @@
+
+## Impact on players
+
+Modding is a significant aspect of the Minecraft community, with players using mods to enhance their gameplay experience. However, Mojang's new update release cycle has had a significant impact on players, particularly those who rely on mods. This section examines the impact of the update cycle on players, including their preferences, confusion, and frustration.
+
+Players have different preferences when it comes to updates. Some prefer small, frequent updates, while others prefer larger, more significant updates. Mojang's new update cycle, which involves pushing out small updates sooner, has been well-received by the majority of vanilla players. However, this new cycle has negatively impacted modders and the players who rely on mods.
+
+The new update cycle has caused confusion and frustration among players who use mods. Modders need time to update their mods to work with the latest version of Minecraft. However, with the new update cycle, modders are struggling to keep up with the updates, leading to mods that are incompatible with the latest version of Minecraft.
+
+Players who use mods are frustrated by the lack of compatibility with the latest version of Minecraft. They are forced to choose between updating their game and losing access to their favourite mods or sticking with an older version of Minecraft and missing out on new features and improvements.
+
+Furthermore, the new update cycle has caused confusion among players who are not familiar with the mods. They may not understand why their favourite mods no longer work, leading to frustration and disappointment.
+
+# Modders' opinions
+
+Modders in the Minecraft modding community have shared their feedback on the impact of Mojang's new update release cycle on modders.
+
+## Positive feedback
+
+Some modders have praised Mojang's new update cycle for providing players with more frequent updates and bug fixes. This, in turn, creates a better gaming experience for players. Additionally, some modders believe that the new update cycle can help modders by providing them with a more stable platform to work with.
+
+One modder states that "Mojang's new schedule is much more flexible in that it allows them to fix bugs and develop features in parallel. No longer will they have to work a few silent months to release a big update all at once, now they can have a more streamlined release process with smaller, more often updates where all experimental content wise changes are offloaded into the experimental datapack instead of only snapshots, which means we can use them sooner."
```

Comment:
This should be Markdown Quotes for consistency with the Curseforge article

```suggestion
> Mojang's new schedule is much more flexible in that it allows them to fix bugs and develop features in parallel. No longer will they have to work a few silent months to release a big update all at once, now they can have a more streamlined release process with smaller, more often updates where all experimental content wise changes are offloaded into the experimental datapack instead of only snapshots, which means we can use them sooner.
```